server/src/models/services/populate.js

```diff
@@ -167,10 +202,362 @@ export default async function ({ models }) {
+      .flatMap(({ id, name_en, name_fr, service_report }) => ({
+        id: `${subject_id}_${id}`,
+        service_id: id,
+        subject_id,
+        name_en,
+        name_fr,
+        value: _.reduce(
+          delivery_channels_keys,
+          (sum, key) =>
+            sum + (_.chain(service_report).sumBy(key).toNumber().value() || 0),
+          0
+        ),
+      }))
+      .filter("value")
+      .sortBy("value")
+      .takeRight(10)
+      .value();
+  };
+
+  const dept_top_application_vol_summary = _.chain(service_rows)
+    .groupBy("org_id")
+    .flatMap(get_top_application_vol_summary)
+    .value();
+  const program_top_application_vol_summary = _.chain(service_rows)
+    .reduce(group_by_program_id, {})
+    .flatMap(get_top_application_vol_summary)
+    .value();
+
+  const gov_services_high_volume_summary = _.chain(service_rows)
+    .groupBy("org_id")
+    .map((services, org_id) => ({
+      id: org_id,
+      subject_id: org_id,
+      total_volume: _.sumBy(services, ({ service_report }) =>
+        _.reduce(
+          delivery_channels_keys,
+          (sum, key) =>
+            sum + _.chain(service_report).sumBy(key).toNumber().value() || 0,
+          0
+        )
+      ),
+    }))
+    // 45,000+ volume is considered "high volume"
+    .reject(({ total_volume }) => total_volume <= 45000)
+    .sortBy("total_volume")
+    .reverse()
+    .value();
+
+  const post_process_website_visits = (data) =>
+    _.chain(data)
+      .filter("website_visits_count")
+      .sortBy("website_visits_count")
+      .takeRight(10)
+      .value();
+
+  const gov_top_website_visits_summary = _.chain(service_rows)
+    .groupBy("org_id")
+    .map((org_services, org_id) => ({
+      id: `gov_${org_id}`,
+      subject_id: org_id,
+      website_visits_count: _.sumBy(
+        org_services,
+        ({ service_report }) =>
+          _.chain(service_report)
+            .sumBy(online_inquiry_count)
+            .toNumber()
+            .value() || 0
+      ),
+    }))
+    .thru(post_process_website_visits)
+    .value();
+
+  const get_top_website_visits_summary = (services, subject_id) =>
+    _.chain(services)
+      .flatMap(({ id, name_en, name_fr, service_report }) => ({
+        id: `${subject_id}_${id}`,
+        subject_id,
+        service_id: id,
+        service_name_en: name_en,
+        service_name_fr: name_fr,
+        website_visits_count:
+          _.chain(service_report)
+            .sumBy(online_inquiry_count)
+            .toNumber()
+            .value() || 0,
+      }))
+      .value();
+
+  const dept_top_website_visits_summary = _.chain(service_rows)
+    .groupBy("org_id")
+    .flatMap(get_top_website_visits_summary)
+    .thru(post_process_website_visits)
+    .value();
+  const program_top_website_visits_summary = _.chain(service_rows)
+    .reduce(group_by_program_id, {})
+    .flatMap(get_top_website_visits_summary)
+    .thru(post_process_website_visits)
+    .value();
```

Comment:
All of the above computations could be broken up into each of files.. let me know any suggestions on organization